maintenance_plan/views/maintenance_plan_views.xml

```diff
@@ -18,10 +18,9 @@
         <field name="arch" type="xml">
             <form string="Maintenance Plan">
                 <sheet>
+                    <widget name="web_ribbon" title="Archived" bg_color="bg-danger" attrs="{'invisible': [('active', '=', True)]}"/>
+                    <field name="active" widget="boolean_button" options="{&quot;terminology&quot;: &quot;archive&quot;}"/>
```

Comment:
I think you should remove this line.

![image](https://user-images.githubusercontent.com/44768500/72450008-e6dde280-37b9-11ea-9b0b-a93e52c0b86a.png)